modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -856,26 +947,61 @@ See xref:6@studio::input-output-structure-transformation-studio-task.adoc[To Def
 [NOTE]
 Schemas with type `Binary` or `Packed` don't allow for line breaks to be detected, so setting `recordParsing` to "lenient"` will only allow long records to be handled, but not short ones. These schemas also currently only work with certain single-byte character encodings (so not with UTF-8 or any multibyte format).
 
-=== Writer Properties
+[[copybook_writer_props]]
+=== Writer Properties (for Copybook)
 
-When defining an output of type copybook there are a few optional parameters you can add to the output directive to customize how the data is written:
+When you define an output of type copybook there are a few optional parameters you can add to the output directive to customize how the data is written:
```

Comment:
More active wording: "...you can add these optional parameters..."

Reply:
done